service/history/nDCActivityReplicator_test.go

```diff
@@ -338,7 +335,8 @@ func (s *activityReplicatorSuite) TestSyncActivity_IncomingScheduleIDLarger_Inco
 	).AnyTimes()
 
 	err = s.nDCActivityReplicator.SyncActivity(context.Background(), request)
-	s.Equal(serviceerrors.NewRetryTask(ErrRetrySyncActivityMsg, namespaceID, workflowID, runID, nextEventID), err)
+	//s.Equal(serviceerrors.NewRetryTask(ErrRetrySyncActivityMsg, namespaceID, workflowID, runID, nextEventID), err)
+	s.NotNil(err)
```

Comment:
uncomment the code?

Reply:
`ErrRetrySyncActivityMsg` was only returned from nDC codepath.  There is no place in the which returns this error now.